component/answer/src/main/java/org/exoplatform/management/answer/operations/AnswerExportTask.java

```diff
@@ -54,7 +63,55 @@ public String getEntry() {
   public static String getEntryPath(String type, String id) {
     return new StringBuilder("answer/").append(type).append("/").append(id).append(FILENAME).toString();
   }
-
+  private List<Category> getAllSubCategories(String path) {
```

Comment:
In the signature of the method getSubCategories(), the parameter categoryId is used i think that we should use the category.getId() as parameter when call this method, in this case an exception is dispalyed in the log file. However, even when we use the category.getPath() as parameter for this method it works perfectly.So, we can use the getSubCategories() with category's path as parameter without need to add a new method.